target/linux/ath79/dts/ar9331_alfa-network_ap121f.dts

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9330.dtsi"
```

Comment:
What about ar9331.dtsi here?

Reply:
You're absolutely right; I've just changed that. :+1: 

I thought it had a AR9330 SoC (so says /proc/cpuinfo), until I opened the case to see it was an AR9331. I guess they're very similar SoCs...

Reply:
Well, have a look into ar9331.dtsi ...